### PR TITLE
preliminary 3.5.0 changelog

### PR DESCRIPTION
**Pull request by Bob-Murphy**
Adds preliminary changelog for 3.5.0 update
version number in code as well as readme is already adjusted

### CODE REVIEW

File: changelog.rst [hunk 1]
```diff
@@ -3,6 +3,103 @@ Changelog
...
+   11th February 2024
+
+.. admonition:: Major
+
+   .. rubric:: :issue:`3075` Weferlingen Map Ports
+
+
+   * [Added] - Complete port for Weferlingen Summer - requires `Global Mobilisation CDLC <https://store.steampowered.com/app/1042220/Arma_3_Creator_DLC_Global_Mobilization__Cold_War_Germany/>`_
+   * [Added] - Complete port for Weferlingen Winter - requires `Global Mobilisation CDLC <https://store.steampowered.com/app/1042220/Arma_3_Creator_DLC_Global_Mobilization__Cold_War_Germany/>`_
+   * In summer and in winter, resistance fighters armed with their R-700 bolt-action rifles trudge through the fields towards ill-prepared Bundeswehr and NVA roadblocks. In an occupied post-WW2 Germany, the citizens are ready to take up arms for freedom, no matter which time of the year.
```
Comment (Tiny-DM) [inline changelog.rst:19]: The summer and The winter

File: changelog.rst [hunk 2]
```diff
@@ -3,6 +3,103 @@ Changelog
...
+   * :issue:`3114` Fixed a bug where remote-controlling players could despawn garrisons.
+   * :issue:`3110` Fixed crew mounting statics while under active attack.
+   * :issue:`3110` Added garbage cleans moving dead soldiers out of their vehicle, which fixes a bug with ACE where seats would be locked if a garbage clean happened.
+   * :issue:`3102` Fixed a bug where the hint would be empty if you tried to purchase vehicles at HQ with enemies nearby.
+   * :issue:`3102` Fixed a bug where refugee missions would spawn on top of people.
+   * :issue:`3102` Fixed Kill the Traitor mission will not succeeding if the traitor surrenders.
+   * :issue:`3102` Fixed Guided launchers being unlockable from large intel.
+   * :issue:`3102` Fixed ASFs climbing endlessly during circling.
+   * :issue:`3102` Added vehicles now have a few more restrictions before being turned into airstrikes, which fixes a few exploits.
+   * :issue:`3102` Added garage now deleting bugged vehicles instead of registering them.
```
Comment (Tiny-DM) [inline changelog.rst:73]: The garage now deletes bugged vehicles instead of registering them.

File: changelog.rst [hunk 3]
```diff
@@ -3,6 +3,103 @@ Changelog
...
+.. admonition:: Bugfixes
+
+   * :issue:`3126` Fixed an exploit where helicopters could retain undercover in enemy airspace.
+   * :issue:`3116` Fixed a minor issue with outpost vehicle spawns.
+   * :issue:`3114` Fixed a bug where remote-controlling players could despawn garrisons.
+   * :issue:`3110` Fixed crew mounting statics while under active attack.
+   * :issue:`3110` Added garbage cleans moving dead soldiers out of their vehicle, which fixes a bug with ACE where seats would be locked if a garbage clean happened.
+   * :issue:`3102` Fixed a bug where the hint would be empty if you tried to purchase vehicles at HQ with enemies nearby.
+   * :issue:`3102` Fixed a bug where refugee missions would spawn on top of people.
+   * :issue:`3102` Fixed Kill the Traitor mission will not succeeding if the traitor surrenders.
```
Comment (Tiny-DM) [inline changelog.rst:69]: Fixed a bug where the Kill the Traitor mission would break if the traitor surrendered.

File: changelog.rst [hunk 4]
```diff
@@ -3,6 +3,103 @@ Changelog
...
+   * :issue:`3091` WS ION - Multiple small changes to the template, mostly vehicles and weapons.
+   * :issue:`3037` RHS ChDKZ - Added additional uniforms and unit templates.
+
+.. admonition:: Bugfixes
+
+   * :issue:`3126` Fixed an exploit where helicopters could retain undercover in enemy airspace.
+   * :issue:`3116` Fixed a minor issue with outpost vehicle spawns.
+   * :issue:`3114` Fixed a bug where remote-controlling players could despawn garrisons.
+   * :issue:`3110` Fixed crew mounting statics while under active attack.
+   * :issue:`3110` Added garbage cleans moving dead soldiers out of their vehicle, which fixes a bug with ACE where seats would be locked if a garbage clean happened.
```
Comment (Tiny-DM) [inline changelog.rst:66]: Garbage cleans now move dead soldiers out of their vehicle, fixing a bug with ACE where seats would be locked if bodies were deleted in a seat.

File: changelog.rst [hunk 5]
```diff
@@ -3,6 +3,104 @@ Changelog
...
+   * :issue:`3102` Fixed a bug where the Kill the Traitor mission would break if the traitor surrendered.
+   * :issue:`3102` Fixed Guided launchers being unlockable from large intel.
+   * :issue:`3102` Fixed ASFs climbing endlessly during circling.
+   * :issue:`3102` Added vehicles now have a few more restrictions before being turned into airstrikes, which fixes a few exploits.
+   * :issue:`3102` Added that the garage now deletes bugged vehicles instead of registering them.
+   * :issue:`3093` Fixed a bug where garrisons would have trouble spawning without buildings.
+   * :issue:`3092` Fixed a check broken in #3031 for patrols.
+   * :issue:`3090` Fixed a tiny bug where garage interactions would cause a logs error.
+   * :issue:`3086` Fixed packed objects (e.g. repair station) not returning money when sold.
+   * :issue:`3079` Fixed a bug where trying to breach whilst undercover will do nothing and an exploit where you could breach for free under certain circumstances.
```
Comment (Tiny-DM) [inline changelog.rst:79]: Can probably truncate this to "fixed two bugs with breaching vehicles".

File: changelog.rst [hunk 6]
```diff
@@ -3,6 +3,104 @@ Changelog
...
+   * :issue:`3053` Fixed an issue where fuel trucks could spawn fuel out of thin air with ACE.
+
+.. admonition:: Groundwork
+
+   * :issue:`3102` Added a system for blacklisting vehicles from the garage.
+
```
Comment (targetingsnake) [inline changelog.rst:85]: if we add #3131 we should mentioned it here or at minor stuff

File: changelog.rst [hunk 7]
```diff
@@ -3,6 +3,104 @@ Changelog
...
+   * :issue:`3088` ACE Hunger - Added bananas! And sunflower seeds and humanitarian rations, but more importantly, bananas.
+   * :issue:`3087` Kat's Advanced Medical (KAM) support has been updated to 2.13.3.
+   * :issue:`3083` Rebel AI will now sometimes be equipped with pistols if they don't have any good primary weapons.
+   * :issue:`3049` APCs in the default Arma 3 factions will now use slat cages to boost their combat effectiveness where able.
+
+ .. admonition:: Template updates
+
+   * :issue:`3091` WS ION - Multiple small changes to the template, mostly vehicles and weapons.
+   * :issue:`3037` RHS ChDKZ - Added additional uniforms and unit templates.
+
```
Comment (targetingsnake) [inline changelog.rst:60]: issue https://github.com/official-antistasi-community/A3-Antistasi/pull/2802 swaped rebel transport truck with cargo truck

**Review by targetingsnake — CHANGES_REQUESTED** [review]

File: changelog.rst [hunk 8]
```diff
@@ -3,6 +3,104 @@ Changelog
...
+   * [Added] - :issue:`2802` GM NVA Desert template
+
+.. admonition:: Minor
+
+   * :issue:`3134` Added hint on old build button in Y menu to help older players find the new construction system. 
+   * :issue:`3115` Rebel and enemy airstrikes will now be shown in the log. 
+   * :issue:`3110` Aircraft can now be accessed from any garage, but cannot be removed unless the player is at an airbase.
+   * :issue:`3102` Seaports will only cut 5% of vehicle price each (changed from 10% each) and now cap at 6 maximum, for a total of a max 30% discount on vehicles.
+   * :issue:`3102` Garage will now delete blacklisted vehicles, so hopefully no more empty "Vehicle" entities.
+   * :issue:`3102` Added the last victim to the hint admins recieve for friendly fire.
```
Comment (jaj22) [inline changelog.rst:49]: recieve -> receive

File: changelog.rst (hunk 5 above)
Comment (Bob-Murphy) [reply changelog.rst:79]: eh, can stay this way. doesn't hurt to point out exploits fixed

**Review by targetingsnake — APPROVED** [review]

**Pull request merged**